src/jekyll/_layouts/showcase/spotlight.liquid

```diff
@@ -182,9 +184,17 @@
   {% if showcase.sections %}
     {% for section in showcase.sections %}
     <section class="wf-showcase__section-container">
-      <h3>{{section.title}}</h3>
-      {{ section.content || markdownify }}
+      <h3 id="{{ section.title | cgi_escape }}">{{section.title}}</h3>
```

Comment:
I really like having `id` attributes so that you can link to specific headings in the page. (Especially when it's a long page.)